src/JustSaying/Messaging/Channels/SubscriptionGroups/SubscriptionGroupSettings.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using JustSaying.AwsTools.MessageHandling;
+
+namespace JustSaying.Messaging.Channels.SubscriptionGroups
+{
+    public class SubscriptionGroupSettings
+    {
+        internal SubscriptionGroupSettings(
```

Comment:
Why `internal`?

Reply:
Because it should be created using the `SubscriptionGroupConfigBuilder`

Reply:
But why does that need to be enforced?

Reply:
So the [validation](https://github.com/justeat/JustSaying/blob/d4e54c10b45838519f8d7e204d5e8b4c3ae436d6/src/JustSaying/Messaging/Channels/SubscriptionGroups/SubscriptionGroupConfigBuilder.cs#L139) runs

Reply:
But we're creating the object there and calling the `Validate()` method before we return it to the caller, so the accessibility doesn't matter in that case. It just seems to prevent the caller for making one themselves separate to the builder syntax using the types themselves if they wanted to.

Reply:
I think the trade off we're making here is for safety over accessibility. Is there any reason we want to allow people to create their own group settings types outside of our builder and bypassing validation? Given that the builders can stand on their own and serve that purpose, I'd suggest that we should err on the side of safety and keep these types internal, forcing users to go through the builder.